addons/binding/org.openhab.binding.nest/src/main/java/org/openhab/binding/nest/discovery/NestDiscoveryService.java

```diff
@@ -0,0 +1,106 @@
+package org.openhab.binding.nest.discovery;
```

Comment:
Every Java file must have a license header. You can run `mvn license:format` on the root of the repo to automatically add missing headers.